events/2022-moveit-community-meeting/index.markdown

```diff
@@ -0,0 +1,47 @@
+---
+author: Sean Kolk
+comments: false
+date: 2022-01-10 10:00:00+00:00
+layout: page
+slug: 2022-moveit-community-meeting
+title: 2022 MoveIt Community Meeting
+---
+
+## 2022 MoveIt Community Meeting
+The 2022 Moveit Community is a time for the maintainers, enthusiasts, and experimenters with MOveit to meet, discuss, and decide the future of MoveIt. The event will be 5 hours with speakers, decision groups, and exhibitions to show MoveIt projects to the community. The event will be held virtually, with details to be determined.
+
+
+### Time and Venue
+
+* Date: Thursday, Feb 24, 2022
+* Venue: (virtual)
+
+### Organizers
+
+* Mark Moll (PickNik)
+* Henning Kayser (PickNik)
+* Tyler Weaver (PickNik)
+
+
+### Program (Tentative)
+
+All times are UTC-6 (US Mountain  Time)
+
+| Opening                             | 08:00:00    |
+| ----------------------------------- | ----------- |
+| MoveIt 2022 highlights              | 08:15:00    |
+| Speaker 1                           | 8:30:00 AM  |
+| Working group break out sessions    | 9:00:00 AM  |
+| Networking session (dedicated time) | 10:00:00 AM |
+| Speaker 2                           | 11:00:00 AM |
+| Working group break out sessions    | 11:30:00 AM |
+| Closing Comments                    | 12:30:00 PM |
+| ----------------------------------- | ----------- |
```

Comment:
I think a consistent style here would look better, and I am not sure we need accuracy down to the second.
```suggestion
| Opening                             |  8:00 AM    |
| ----------------------------------- | ----------- |
| MoveIt 2022 highlights              |  8:15 AM   |
| Speaker 1                           | 8:30 AM  |
| Working group break out sessions    | 9:00 AM  |
| Networking session (dedicated time) | 10:00 AM |
| Speaker 2                           | 11:00 AM |
| Working group break out sessions    | 11:30 AM |
| Closing Comments                    | 12:30 PM |
| ----------------------------------- | ----------- |
```

Reply:
Agree this is a good change.  Made me think of this: https://xkcd.com/2553/